src/types/index.ts

```diff
@@ -135,4 +135,18 @@ export interface ErrorContextType {
 export interface SyncOrUpdateContextType {
   syncOrUpdate?: string;
   setSyncOrUpdate: (message: string) => void;
+}
+
+/**
+ * interface type request
+ */
+export interface Request {
```

Comment:
There is some duplication between the files as we have the Request interface here, another request interface in employeeVacationRequests.tsx which is very similar, and also the VacationRequests interface from the generated files, which again is almost identical. So hopefully this interface can be removed and all other uses of the type can reference the VacationRequests from the generated files.